var/spack/repos/builtin/packages/httpress/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Httpress(MakefilePackage):
+    """High performance HTTP server stress & benchmark utility."""
+
+    homepage = "https://bitbucket.org/yarosla/httpress/wiki/Home"
+    hg       = "https://bitbucket.org/yarosla/httpress"
+
+    version('develop')
+    version('1.1.0', revision='1.1.0', preferred=True)
```

Comment:
I don't think `preferred=True` is needed here, Spack already defaults to the newest stable release